taichi/codegen/codegen.cpp

```diff
@@ -131,11 +134,15 @@ std::vector<LLVMCompiledData> KernelCodeGen::compile_kernel_to_module() {
   if (!kernel->is_evaluator) {
     worker.flush();
   }
+  auto linked = tlctx->link_compile_data(std::move(data));
+  std::vector<LLVMCompiledData> linked_data;
```

Comment:
IMHO a more appropriate name for LLVMCompiledData is actually LLVMCompiledKernel? :P

Reply:
I will split LLVMCompiledData to `LLVMCompiledKernel` and `LLVMCompiledTask` in the future, because we only need `used_snode_trees` and `` when compiling tasks and linking, and after we link the tasks to a kernel, we don't need them anymore.